src/exec/Makefile

```diff
@@ -17,6 +17,19 @@ else
 	@echo "CARGO (debug) => exec"
 endif
 
+format: $(SRC_FILES)
+	@output=$$(cargo fmt -- --check 2>&1); retval=$$?; \
+		if [[ $$retval -eq 1 ]]; then \
+			$(ECHO) "$$output"; cargo fmt; $(ECHO) "$(GREEN)\nRust code format corrected.$(NO_COLOR)"; \
+		fi
+
+format-check: $(SRC_FILES)
+	@output=$$(cargo fmt -- --check 2>&1); retval=$$?; \
+		if [[ $$retval -eq 1 ]]; then \
+			$(ECHO) "$(RED)\nRust code format error detected:$(NO_COLOR)"; $(ECHO) "\n$$output"; \
+			$(ECHO) "\nTo get rid of the format warnings above, run $(CYAN)"make format"$(NO_COLOR) to correct"; \
+		fi
```

Comment:
The format targets in `exec/Makefile` and `libos/Makefile` have some duplicated code (the format command and the warning messages). I think it is better to define functions in `sgxenv.mk` so that we do not repeat ourselves in the Makefiles in sub-projects.

Another suggestion is to change "Rust code format error detected"  to "Some format issues of Rust code are detected".

Reply:
Done.